core/iwasm/compilation/aot_compiler.h

```diff
@@ -163,7 +163,15 @@ check_type_compatible(uint8 src_type, uint8 dst_type)
 #define POP_V128(v) POP(v, VALUE_TYPE_V128)
 #define POP_FUNCREF(v) POP(v, VALUE_TYPE_FUNCREF)
 #define POP_EXTERNREF(v) POP(v, VALUE_TYPE_EXTERNREF)
-
+#if WASM_ENABLE_GC != 0
+#define POP_REF(v) POP(v, VALUE_TYPE_GC_REF)
+#else
+#define POP_REF(v)                                                        \
+    do {                                                                  \
+        LOG_DEBUG("enable_gc=true when CMake flag are not set properly"); \
```

Comment:
How about `bh_assert(!"should not POP_REF when WASM_ENABLE_GC macro isn't enabled");`